local/modules/Tinymce/Tinymce.php

```diff
@@ -41,6 +43,16 @@ public function postActivation(ConnectionInterface $con = null)
     {
         $fileSystem = new Filesystem();
 
+        //Check for environment
+        if ($env = $this->getContainer()->getParameter('kernel.environment')) {
+            //Check for backward compatibility
+            if ($env !== "prod") {
```

Comment:
In the config.php we never add _dev to the environment so for exemple:

you have an environment TEST_THELIA, in dev mode you TEST_THELIA_dev but tinymce must not be dependent to dev / prod so it only use TEST_THELIA. So files will be copied in ```media/TEST_THELIA```